packages/react-dom/src/ReactDOMSharedInternals.js

```diff
@@ -39,14 +40,15 @@ const DefaultDispatcher: HostDispatcher = {
   preinitModuleScript: noop,
 };
 
-const Internals: InternalsType = {
-  usingClientEntryPoint: false,
-  Events: (null: any),
-  ReactDOMCurrentDispatcher: {
-    current: DefaultDispatcher,
-  },
+const Internals: ReactDOMInternals = {
+  d /* ReactDOMCurrentDispatcher */: DefaultDispatcher,
+  p /* currentUpdatePriority */: NoEventPriority,
   findDOMNode: null,
-  up /* currentUpdatePriority */: NoEventPriority,
+  usingClientEntryPoint: false,
 };
 
+// if (__DEV__) {
+//   (Internals: any).usingClientEntryPoint = false;
+// }
```

Comment:
Why is this commented out?

Reply:
I was sloppy. I can land removing it but I’ve also already removed it in the next change which moves around all the entry points.